File: FontLocationDemo/FontLocationDemo/ViewController.m
//
//  ViewController.m
//  FontLocationDemo
//
//  Created by shenzhenshihua on 2017/9/7.
//  Copyright © 2017年 shenzhenshihua. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UITextView *myText;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [self testFontName];
    
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)testFontName {
    for (NSString *familyName in [UIFont familyNames]) {
        NSLog(@"familyName：%@", familyName);
        for (NSString *fontName in [UIFont fontNamesForFamilyName:familyName]) {
            NSLog(@"fontName:%@", fontName);
        }
    }
}
- (IBAction)btnAction:(UIButton *)sender {
    NSArray * fonts = @[@"MFJinHei_Noncommercial-Regular",@"MFTongXin_Noncommercial-Regular",@"MFZhiHei_Noncommercial-Regular",@"Gaspar"];
    [self.myText setFont:[UIFont fontWithName:fonts[sender.tag - 1000] size:17]];

}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: FontSwiftDemo/FontSwiftDemo/ViewController.swift
//
//  ViewController.swift
//  FontSwiftDemo
//
//  Created by shenzhenshihua on 2017/6/16.
//  Copyright © 2017年 shenzhenshihua. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var gaspar: UIButton!
    @IBOutlet weak var mftongxin: UIButton!
    override func viewDidLoad() {
        super.viewDidLoad()
        self.download(path: "cakkkk")
        // Do any additional setup after loading the view, typically from a nib.
    } 

    
    
    func download(path:String) {
    
        let store = Store()
        _ = store.write(any: "hello", path: "hehe")
        
        let somes = store.read(path: "hehe")
        
        print(somes)
//       print(store.returnPath(path: path))
        
        

    }
    
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

